install/ui/src/freeipa/widget.js

```diff
@@ -2509,6 +2509,9 @@ IPA.custom_checkboxes_widget = function(spec) {
 
     var that = IPA.checkboxes_widget(spec);
 
+    that.set_value_to_lowercase = spec.set_value_to_lowercase === undefined
+                                    ? true : spec.set_value_to_lowercase;
```

Comment:
Would be better if the default behavior was 'no lower casing' -> change to false and then adjust usages of the widget.

Reasoning: no formatting seems more nutural